static/js/publisher/market/market.js

```diff
@@ -52,17 +52,15 @@ function initFormNotification(formElId, notificationElId) {
 
 
 function initForm(config, initialState, errors) {
-  // if there are errors mark fields as invalid
+  // if there are errors focus first error
   if (errors && errors.length) {
-    errors.forEach((error) => {
-      if (error.code === 'invalid-field') {
-        const name = error.extra.name;
-        if (name) {
-          const input = document.querySelector(`[name=${name}]`);
-          input.closest('.p-form-validation').classList.add('is-error');
-        }
-      }
-    });
+    // find input with error or error notification and scroll it into view
+    const errorInput = document.querySelector('.is-error input')
+      || document.querySelector('.p-notification--negative');
+
+    if (errorInput) {
+      errorInput.scrollIntoView();
```

Comment:
This will probably cause errors in older browsers, but worth going - maybe a check around it to check scrollIntoView is defined? Not the worst thing if the view doesn't scroll to the input.